### DISCUSSION TITLE
마이크로소프트 365 개발자 테넌트 생성 오류

### DISCUSSION THREAD

**Discussion by jakysn** [fusiondevkr/pl900#80] [1 Q&A] [answered]
마이크로소프트 365 개발자 테넌트 생성시 아래와 같은 문구가 뜨는데 어떤 이유인지 알 수 있을까요?

Due to the high demand, we may not be able to provide everyone with a configurable sandbox. We are doing our best to resolve this issue. We thank you for your understanding.

**Comment by sesme100** [comment, marked as answer]
안녕하세요 
혹시 6시간이 지난 지금도 동일한 문제가 생기나요? 다시 한번 M365 개발자 테넌트 생성 해보시고 문제가 생기시면 안내 부탁드립니다. 
그리고 에러메세지만 봐서는 파악이 안되는데요. M365개발자 생성 어느 단계에서 에러메세지가 나왔는지 알려주시면 감사하겠습니다.

**Reply by jakysn** [reply]
![image](https://user-images.githubusercontent.com/23229104/142962329-9aeaa8a6-9069-4f04-855d-845d554ef552.png)

Set up your Microsoft 365 E5 sandbox 화면에서 테넌트 계정 생성을 하려고 하니 해당 오류가 발생합니다.
마이크로소프트 측에서 개발자 테넌트 계정 수를 제한한건 아닌지 의문입니다.

**Reply by justinyoo** [reply]
아, 이건 테넌트 계정은 이미 만들어진 거구요, 위 스크린샷처럼 미리 뭔가를 가상의 팀을 만들어 구성해 준다는 얘긴데, 그냥 아래 "Configurable Sandbox" 선택하셔도 됩니다.
